lib/granite/base.rb

```diff
@@ -0,0 +1,27 @@
+require 'active_data/model'
+require 'active_data/model/primary'
+require 'active_data/model/lifecycle'
+require 'active_data/model/associations'
+
+require 'granite/action/translations'
+require 'granite/action/represents'
+
+module Granite
+  module BaseConcern
+    extend ActiveSupport::Concern
+
+    include ActiveSupport::Callbacks
+    include ActiveData::Model
+    include ActiveData::Model::Representation
+    include ActiveData::Model::Associations
+    include ActiveData::Model::Dirty
+    include ActiveModel::Validations::Callbacks
+
+    include Granite::Action::Translations
+    include Granite::Action::Represents
+  end
+
+  class Base
+    include BaseConcern
+  end
```

Comment:
if we have `Base` class then the value of having `BaseConcern` diminishes. With the current implementation, all the AD structures will be `is_a?(Base) # true` while `Granite::Action.is_a?(Base) # false` which creates an opportunity for mistakes. Lets have `class Base` only in this case.